src/ucp/wireup/select.c

```diff
@@ -1850,6 +1871,34 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
         bw_info.criteria.local_md_flags |= UCT_MD_FLAG_INVALIDATE_RMA;
     }
 
+    /* Find an AM lane that needs to be segregated from RMA accesses */
+    for (lane_desc_idx = 0; lane_desc_idx < select_ctx->num_lanes;
```

Comment:
maybe use a helper var for lane_desc and use ucs_carray_for_each
`ucs_carray_for_each(lane_desc, select_ctx->lane_descs, select_ctx->num_lanes)`
to simplify the code

Reply:
fixed